.github/workflows/ci.yml

```diff
@@ -0,0 +1,221 @@
+---
+name: "CI"
+on:
+  - "push"
+  - "pull_request"
+
+jobs:
+  black:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_CHATOPS_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Set up Python"
+        uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - name: "Install Python Packages for Setup"
+        run: "pip install invoke poetry"
+      - name: "Install Python Packages for Linting"
+        run: "poetry install"
+      - name: "Linting: black"
+        run: "poetry run invoke black"
```

Comment:
```suggestion
        run: "invoke black"
```

Reply:
@jvanderaa @chadell please check the following link:
https://github.com/nautobot/nautobot-plugin-chatops/runs/3500131352?check_suite_focus=true

Having just `invoke black` after `poetry install` is not enough. You have to start a shell or use `poetry run xxxx`.